<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="test1"
           tests="8"
           failures="1"
           errors="0"
           skipped="7"
           time="0.010">
  <properties />
            <testcase name="setUp" time="0.010" classname="com.first.testcases.NewTest">
                <failure type="java.lang.AssertionError"
                  message="error:初始化报错，不执行后面的代码"
              >
<![CDATA[java.lang.AssertionError: error:初始化报错，不执行后面的代码
  org.testng.Assert.fail(Assert.java:96)
  com.first.testcases.NewTest.setUp(NewTest.java:25)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
  org.testng.TestRunner.beforeRun(TestRunner.java:529)
  org.testng.TestRunner.run(TestRunner.java:497)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
  org.testng.SuiteRunner.run(SuiteRunner.java:364)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
  org.testng.TestNG.runSuites(TestNG.java:1049)
  org.testng.TestNG.run(TestNG.java:1017)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
            <testcase name="tearDome" time="0.0" classname="com.first.testcases.NewTest">
          <skipped />
    </testcase>
            <testcase name="setup" time="0.0" classname="com.first.testcases.NewTest">
          <skipped />
    </testcase>
            <testcase name="tearDome" time="0.0" classname="com.first.testcases.NewTest">
          <skipped />
    </testcase>
            <testcase name="setup" time="0.0" classname="com.first.testcases.NewTest">
          <skipped />
    </testcase>
            <testcase name="testDome" time="0.0" classname="com.first.testcases.NewTest">
          <skipped />
    </testcase>
            <testcase name="f" time="0.0" classname="com.first.testcases.NewTest">
          <skipped />
    </testcase>
            <testcase name="f2" time="0.0" classname="com.first.testcases.NewTest">
          <skipped />
    </testcase>
  
  
</testsuite>
